Update direct-routing-plan-media-bypass.md (#2638)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan-media-bypass.md

### Requirements for using media processors

> Media Processors are always in the media path for voice applications and for Web cleints (for exampe, Teams cleint in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.

Reviewer: ```suggestion
Media Processors are always in the media path for voice applications and for Web clients (for example, Teams clients in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
```

## Client endpoints supported with media bypass

Media bypass is supported with all Teams endpoints.

> Note for webcleints (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even of it started as a bypass call. This happens automatically and does not require any actions on the administrator. 

Reviewer: ```suggestion
> [!NOTE]
> The call will be converted to non-bypass even if it started as a bypass call for web clients using the Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox. This happens automatically and does not require any actions from the administrator. 
```
